src/vinyldns/client.py

```diff
@@ -565,6 +565,40 @@ def list_record_sets(self, zone_id, start_from=None, max_items=None, record_name
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListRecordSetsResponse.from_dict(data)
 
+    def global_list_record_sets(self, start_from=None, max_items=None, record_name_filter=None,
```

Comment:
For consistency with other clients we added this to, could you rename this `search_record_sets`?

Reply:
Updated the method name.